sdk/AzSdk.reference.props

```diff
@@ -0,0 +1,36 @@
+<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
Why is this needed?

Reply:
It contains $(SdkTargetFx) which is being used in the managementplane .csproj as TargetFramework

Reply:
Again I think this is from the management toolset and shouldn't be here. These projects shouldn't be getting built from our pipelines they should still be getting built from the mgmt pipelines but you might need to make some adjustments to ensure that still happens.

Reply:
Ok, I'll do that.